client/src/Movies/MoviePoster.js

```diff
@@ -0,0 +1,10 @@
+const Poster = [
+    {id: 0, img: url('../Img/0_godfather.jpg')},
```

Comment:
Suggest making the value of img in all quotes like so ```"url('../Img/0_godfather.jpg')"``` otherwise an error will come up.